### ISSUE TITLE
CRITICAL: Can not call software version for diamond

### ISSUE THREAD

**Issue by ZhangDengwei** [biobakery/biobakery#20]
Please direct your questions/bugs to [bioBakery Support Forum](https://forum.biobakery.org/.) under specific categories..
Hi,

I am using humann3 to deal with my dataset, and I encountered the following error:
`09/03/2020 01:27:17 PM - humann.utilities - DEBUG: Check software, metaphlan, for required version, 3.0
09/03/2020 01:27:18 PM - humann.utilities - INFO: Using metaphlan version 3.0
09/03/2020 01:27:18 PM - humann.utilities - DEBUG: Check software, bowtie2, for required version, 2.2
09/03/2020 01:27:18 PM - humann.utilities - INFO: Using bowtie2 version 2.3
09/03/2020 01:27:18 PM - humann.humann - INFO: Search mode set to uniref90 because a uniref90 translated search database is selected
09/03/2020 01:27:18 PM - humann.utilities - DEBUG: Check software, diamond, for required version, 0.9.24
09/03/2020 01:27:18 PM - humann.utilities - CRITICAL: Can not call software version for diamond
`
Then I tested my `diamond` with the command "diamond --version" and "diamond version", both could work, as
`$ diamond --version
diamond v0.9.26.127 | by Benjamin Buchfink <buchfink@gmail.com>
Licensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>
Check http://github.com/bbuchfink/diamond for updates.

diamond version 0.9.26
`

Can you help me out of this problem? Thanks!

**Comment by ZhangDengwei** [comment]
version 0.9.26 is incompatible with huamnn3, the latest version can work well

**Comment by annotatebio** [comment]
Unfortunately, I stumble upon exactly the same problem. I have installed HUMAnN via `pip install humann` and I got the same error:

```bash
CRITICAL ERROR: Can not call software version for diamond
```

I tried to install version of Diamond mentioned in the `README.md` by using `pip install diamond=0.9.36` but this verions is not available via pip neither via Diamond repository. Upgrading Diamond to the latest version (4.0.515) via `pip install diamond`  did not solve the problem.

Help highly appreciated!
